src/aes_hash.rs

```diff
@@ -213,14 +213,16 @@ impl Hasher for AHasher {
     }
 }
 
-#[cfg(feature = "specialize")]
-pub(crate) struct AHasherU64 {
+/// A specialized hasher for only primitives <= 64 bits.
```

Comment:
If these are public, they should have a bit more docs. It is probably worth enumerating the types that this works with. (Notably `usize` is not one, as it can technically be 128 bits on some exotic systems)